src/opm/output/eclipse/Summary.cpp

```diff
@@ -2230,7 +2230,11 @@ static const std::unordered_map< std::string, Opm::UnitSystem::measure> single_v
   {"FGIPL"    , Opm::UnitSystem::measure::gas_surface_volume },
   {"FGIPG"    , Opm::UnitSystem::measure::gas_surface_volume },
   {"FPR"      , Opm::UnitSystem::measure::pressure },
-
+  {"FGCDI"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FGCDM"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FWCD"     , Opm::UnitSystem::measure::gas_surface_volume },
+  {"FWIPG"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FWIPL"    , Opm::UnitSystem::measure::liquid_surface_volume }
```

Comment:
Tiny nit, but could you put a trailing comma on the last entry here please?  That way, it's slightly easier to add new entries when we need to.